exercicios/para-casa/entregas/mabia/resolucao.js

```diff
@@ -8,8 +8,22 @@ class InvoiceItem {
   getID() {
     return this.id;
   }
+  getDesc() {
+    return this.description;
+  }
+  getQuantity() {
+    return this.quantity;
+  }
+  getUnitPrice() {
+    return this.price;
+  }
```

Comment:
Neste caso, faltou o setQuantity, setUnitPrice, getTotal e o toString, como estava descrito no exercício.

Reply:
getPrice = () => this.Price;

  setPrice = (newPrice) => (this.Price = newPrice);


  getTotal = () => this.Price * this.quantity;

  toString = () =>
  
       `id = ${this.id},
        description = ${this.description},
        quantity = ${this.quantity},
        unitPrice = ${this.Price}`;

}